pkg/operator/ceph/nfs/nfs.go

```diff
@@ -46,17 +47,18 @@ const (
 var updateDeploymentAndWait = opmon.UpdateCephDeploymentAndWait
 
 type daemonConfig struct {
-	ID              string              // letter ID of daemon (e.g., a, b, c, ...)
-	ConfigConfigMap string              // name of configmap holding config
-	DataPathMap     *config.DataPathMap // location to store data in container
+	ID                  string              // letter ID of daemon (e.g., a, b, c, ...)
+	ConfigConfigMap     string              // name of configmap holding config
```

Comment:
How is this configmap related to the RADOS object where the config is stored? If the RADOS config is updated, do we get events for that to restart the daemon as well?

Reply:
The RADOS config contains additional config. This config is needed in order for Ganesha to find the RADOS config(s).

Reply:
Is this something worth documenting? Or maybe an overhaul of the [ceph nfs](https://rook.github.io/docs/rook/latest/ceph-nfs-crd.html) doc is better left for a separate PR.

Reply:
This isn't something that users should be touching, so I don't think it's worth documenting. It's what allows us to run the NFS servers with a generated config that doesn't get made in an init container that needs to run the Rook binary.

Reply:
one more question... When is the configmap updated? When exports are added/removed?

Reply:
Only when the CephNFS CR is updated AFAIK.